content/events/subscribe-to-events/developer-guides/query-app-events-enduser/_index.en.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Query app events as end user
+linktitle: Query app events - end user
+description: Developer guide on how to query app events as an end user or end user system
+weight: 40
+toc: true
+---
+
+{{% notice warning %}}
+Altinn Events enables event driven integration patterns, designed specifically to __avoid__ the need for
+continuously requesting resources, also known as 'polling'. <br/> <br/>
+However, in order to allow for smooth and incremental architectural migrations,
+the Events API also provides an HTTP API for scheduled requests of the same event data you hopefully
+will receive via webhooks in the future.
+{{% /notice %}}
+
+
+## Endpoint
```

Comment:
We don't want new implementations to use this endpoint. Rather the query-events endpoint. Let's highlight this.. Soon to be deprecated warning

Reply:
Will be fixed at a later point in time when subscribe scope is available to those using this endpoint today